Update styles

johnpeart opened the pull request:
This pull request:

- changes the base styles for the tool to match the wider [mentormatching.online](//mentormatching.online) domain
- removes the templates for privacy and cookies, because these are now at  [mentormatching.online/software](//mentormatching.online/software)

A separate branch will look to remove the Jekyll code from this repo.

johnpeart committed:
Update page styles

Various style changes to match with www.mentormatching.online styles

johnpeart committed:
Remove templates and update routes

I've moved the docs to the main [mentormatching.online](//mentormatching.online) Jekyll deployment, so these templates are no longer necessary.

This should allow us to separate concerns, and remove the Jekyll code from this repo entirely.

jonodrew commented on mentor_match_web/app/main/routes.py:
```diff
@@ -37,12 +37,12 @@ def index():
 @main_bp.route("/cookies")
 def cookies():
-    return redirect("http://docs.mentoring.service.civilservice.lgbt/cookies/")
+    return redirect("http://mentormatching.online/cookies/")
```
Should these be https?

johnpeart replied:
Good catch!
Fixed in commit [0265462](https://github.com/mentor-matching-online/mentor-match/pull/12/commits/026546244d8577f15bba71d0d18cf3d1f26caf4f)

johnpeart committed:
Change http:// to https://

Changed redirects to https:// per [this comment](https://github.com/mentor-matching-online/mentor-match/pull/12#discussion_r1298463557)
